bobcat/bb-cumber/src/main/java/com/cognifide/qa/bb/cumber/Bobcumber.java

```diff
@@ -80,12 +87,19 @@ public Bobcumber(Class<?> clazz) throws InitializationError, IOException {
         PrintWriter writer = new PrintWriter(file, CharEncoding.UTF_8);
         writer.close();
       }
+      if (clazz.getName().equals(RERUN_FAILED_TESTS_CLASS_NAME) && !anyTestFailed()) {
```

Comment:
Why should we flip operands? After we detect that we are running differently than Rerun test, it doesn't matter if feature file is empty or not